firebase-vscode/webviews/SidebarApp.tsx

```diff
@@ -134,6 +135,10 @@ export function SidebarApp() {
     );
   }
 
+  function sendMessageToTerminal() {
```

Comment:
Is this only used once? Might as well just make it an anonymous function and put it inline?

Reply:
Changed to inline function in an upcoming commit!